docker/config_template.yaml

```diff
@@ -74,8 +76,8 @@ persistence:
                 connectAttributes:
                     tx_isolation: 'READ-COMMITTED'
                 {{- end }}
-                maxConns: {{ default .Env.SQL_VIS_MAX_CONNS "20" }}
-                maxIdleConns: {{ default .Env.SQL_VIS_MAX_IDLE_CONNS "20" }}
+                maxConns: {{ default .Env.SQL_VIS_MAX_CONNS "10" }}
```

Comment:
curious, what was the reason for this change?

Reply:
there is not many calls in visibility side